plugins/__init__.py

```diff
@@ -70,6 +70,14 @@ def net_to_nm(network_addr: "string") -> "string":
 
 @plugin
 def ipnet(addr: "ip::cidr_v10", what: "string") -> "string":
+    """
+    Return the ip, prefixlen, netmask or network address of the CIDR
+
+    :param ip: The IP address of `addr` object
```

Comment:
This is not according to convention, `:param : ` is  used to refer to a parameter of the plugin. 

The explanation of the possible values for  ':param what` has to be explained in another way.

This is restructured text, so for markup, you can look at https://github.com/ralsina/rst-cheatsheet/blob/master/rst-cheatsheet.rst or https://devguide.python.org/documenting/

to see how it is rendered, look at https://docs.inmanta.com/community/dev/reference/modules/std.html#std.password